cypress/integration/Signup.feature

```diff
@@ -70,3 +77,29 @@ Feature: Signup
         # cypress seems t choke on being redirected off our site though
         # When I logout
         #Then I am taken to the post logout url
+
+    @focus
+    # this scenario must run after the user has saved their details for the first time
```

Comment:
just a thought - this comment suggests there needs to be a `background` section to do this. I guess not practical yet, but does need some thought.

Reply:
Yes, that would be good. The whole of the "Valid user details" test needs to run for this later test to work. Another option would be to (for now) move it into its own feature which runs after signup (with the other post-signup features). At the moment, we don't have a way for signup to be a background, we are relying on test order.

Reply:
@nickdavis2001 could we think about stitching signup into the front of tests, and having a way to put the back-end into a state where the user has signed up, instead of relying on test order?

Reply:
My gut feel (though willing to be persuaded otherwise :) ) is that stitching is a bit of a hack to make things work with both Signed-up user or standard seeded user. Feels like a background step might be better here. Agreed its not ideal that these scenarios are order sensitive, and some refactoring here wouldn't hurt, long term.

Reply:
@nickdavis2001 Shall I raise an issue to investigate this rather than it block this PR?

Reply:
Background step applies to all scenarios.  I'd be inclined to do this in another PR and not block this one.